superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -424,6 +432,12 @@ export const controls = {
   },
 
   groupby: groupByControl,
+  dimension: Object.assign({}, groupByControl, {
```

Comment:
Further up you are doing 
```
{
       ...this.props.value,
       [ctrl]: value,
}
```
here you are using `Object.assign()` instead of 
```
{
   ... groupByControl,
  label: t('Dimension'),
  description: t('Select a dimension'),
  multi: false,
  default: null,
}
```

Reply:
yeah let's move towards spread operator instead of `Object.assign`, we should eventually add a linting rule for this